lib/client_routes.ex

```diff
@@ -0,0 +1,15 @@
+defmodule HTTPServer.Routes.ClientRoutes do
+  alias HTTPServer.Request
+  alias HTTPServer.Handlers.NotFound
+
+  @behaviour HTTPServer.Routes
+
+  @routes %{}
+  @impl HTTPServer.Routes
+  def fetch_route(path) do
+    Map.fetch(@routes, path)
+  end
```

Comment:
This code is repeated throughout all implementations, and its identical, and looks like it would never change.  Should it be implemented only once?  Why would a user have the responsibility of including this kind of code in a configuration?